state-chain/traits/src/lib.rs

```diff
@@ -144,7 +144,7 @@ impl<Id, Amount> From<(Id, Amount)> for Bid<Id, Amount> {
 }
 
 #[derive(PartialEq, Eq, Clone, Debug, Decode, Encode)]
-pub enum VaultStatus<ValidatorId> {
+pub enum VaultRotationStatusOuter<ValidatorId> {
```

Comment:
```VaultRotationStatus``` feels like a better name for it since the status of the vault is in context of rotation. However, ```VaultRotationStatus``` is already used inside the threshold sig pallet which is converted to the the outer status type when queried by the validator. Thats why it seemed ```VaultRotationStatusOuter``` was a better name for it.

Reply:
If we're going to rename it, I think renaming it to be about the "key" and not the vault would be more accurate now. It's actually about key rotation, as vaults are chain specific now and keys are not

Reply:
renamed to ```KeyRotationStatus```

Reply:
actually did a wider rename from from vault -> for various storage, variable and function names.